projects/core/src/auth/services/user-authentication/user-authentication-token.service.ts

```diff
@@ -64,4 +68,20 @@ export class UserAuthenticationTokenService {
       .post<UserToken>(url, params, { headers })
       .pipe(catchError((error: any) => throwError(error)));
   }
+
+  revokeToken(userToken: UserToken): Observable<{}> {
```

Comment:
Since the service is already a "token" service, I'd rather go with just `revoke`. But what you've done is more aligned with what we already have here.

Reply:
No problem,  I changed it to `revoke`.

Reply:
No problem, I changed it to revoke.